_install-and-configure/additional-plugins/ingest-attachment-plugin.md

```diff
@@ -0,0 +1,228 @@
+---
+layout: default
+title: Ingest-attachment plugin
+parent: Installing plugins
+nav_order: 20
+
+---
+
+# Ingest-attachment plugin
+
+The Ingest-attachment plugin enables OpenSearch to extract content and other information from files using the Apache text extraction library [Tika](https://tika.apache.org/).
+Supported document formats include PPT, PDF, RTF, ODF and many more ([Tika Supported Document Formats](https://tika.apache.org/2.9.2/formats.html)).
+
+The input field must be a base64-encoded binary.
+
+## Installing the plugin
+
+Install the `ingest-attachment` plugin using the following command:
+
+```sh
+./bin/opensearch-plugin install ingest-attachment
+```
+
+## Attachment processor options
+
+| Name | Required | Default | Description |
+| :--- | :--- | :--- | :--- |
+| `field` | yes | - | The field to get base64 encoded binary from. |
+| `target_field` | no | attachment | The field that holds the attachment information. |
+| `properties` | no | all properties | An array of properties, which should be stored. Can be `content`, `language`, `date`, `title`, `author`, `keywords`, `content_type`, `content_length`. |
+| `indexed_chars` | no | `100_000` | The number of character used for extraction to prevent fields from becoming to large. Use `-1` for no limit. |
```

Comment:
```suggestion
| `indexed_chars` | No | `100_000` | The number of characters used for extraction to prevent fields from becoming too large. Use `-1` for no limit. |
```